Improved env var docs a bit. (#1913)
wasp-lang/wasp: web/docs/advanced/deployment/manually.md

The command above will build the web client and put it in the `build/` directory in the `web-app` directory.

> This is also the moment to provide any additional env vars for the client code, next to `REACT_APP_API_URL`. Check [env vars docs](../../project/env-vars#client-env-vars-1) for more details.

Reviewer: ```suggestion
Use the build command to provide any additional env vars for the client code (other than `REACT_APP_API_URL`). Check the [env vars docs](../../project/env-vars#client-env-vars-1) for more details.
```
Reasonings:
- "This is" -> "Use the build command" - Better to call things by their name than relying on previous context.
- "The" - I guess this one is pretty clear on its own. :smile: 
- The parentheses I can't explain but looks nicer to me.

Author: - I missed the "the", thanks.
- "This is" -> "Use the build command": not exactly the same, I was referring to the moment at which we are currently ("this is also the moment"), not the build command, so you somewhat changed the semantics of the sentence.

I don't really mind small changes like this, why not, but this one kind of breaks the flow for me -> if you start reading a bit up and read from there, before it was reading ok to me: "Also, this is the moment to provide any additional env vars" -> exactly what I wanted to say. But now, it is "Use the build command" -> a bit weird, starts out of nowhere, also "use" the "command" is slightly weird -> ok, not terribly, but it doesn't emphasize that at this very moment they need to specify client env vars, this is it.

I can add "the", sure, but the rest seems to not be communicating what I wanted to commnicate.

cp fly.toml ../../
```

> Next, let's add a few more environment variables that will be available to the server code:

Reviewer: It's better to stick to the present tense and addressing the user directly:
```suggestion
Next, add a few more environment variables for the server code.
```
